Uebersicht/UBAppDelegate.m

```diff
@@ -166,6 +200,11 @@ - (void)startUp
 
 - (void)shutdown:(Boolean)keepAlive
 {
+    if (shuttingDown) {
```

Comment:
this one is merged as a separate PR already :)

Reply:
Rebased 😄